### ISSUE TITLE
VDL trip

### ISSUE THREAD

**Issue by dolson334** [dolson334/bantor#377]
I am going to VDL Sept 5-8. I am going to demo the site to Ed on that Thursday. Which steps do you think we need to complete before then? make the list here then Ill add to it and make the ticket. 

- new Story pictures
- new Blog pictures
- new fishing report pictures

**Comment by bschmitz9** [comment]
@dolson334 we should be done completely by then:

Everything in site completed
Bugs taken care of the best we can
Info page finalized and working
Demo site set up with real text so it looks like a real website
Up on GoDaddy. 

Everything but reservations done by then. No reason we shouldn't be done at that point
